contracts/Vault.sol

```diff
@@ -358,24 +407,35 @@ contract Vault is
         _unsponsor(_to, _ids);
     }
 
+    function harvest() external onlyRole(HARVESTOR_ROLE) {
```

Comment:
I have to be honest, I don't like the name "harvest", to be "harvest" and "claim" feel like the same thing. Can't we rename this to "withdrawPerformanceFee"?

Reply:
fixed